doc/docs/modules/ROOT/pages/gds.adoc

```diff
@@ -54,17 +51,17 @@ df = spark.read.format("org.neo4j.spark.DataSource") \
 ----
 
 [NOTE]
-If you get a "Graph Already Exists" error, take a look at this xref:faq.adoc#graph-already-exists[FAQ].
+If you get a `A graph with name [name] already exists` error, take a look at this xref:faq.adoc#graph-already-exists[FAQ].
 
 [NOTE]
-**Ensure partitions is set to 1.  You do not want to execute this query in parallel, only once.**
+**Ensure that option `partitions` is set to 1. You do not want to execute this query in parallel, it should be executed only once.**
 
 [NOTE]
-**When you use stored procedures, you must include a RETURN clause**
+**When you use stored procedures, you must include a `RETURN` clause.**
 
-=== Run a GDS Analysis and Stream the Results Back
+=== Run a GDS analysis and stream the results back
 
-To run an analysis, the result is just another Cypher query, executed as a spark read from Neo4j.
+The following example shows how to run an analysis and get the result as just another Cypher query, executed as a Spark read from Neo4j.
```

Comment:
@conker84, please, take a look at this sentence. It was unclear, so I've revised it.